airbyte-commons/src/test/java/io/airbyte/commons/bytes/ByteUtilsTest.java

```diff
@@ -0,0 +1,19 @@
+package io.airbyte.commons.bytes;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ByteUtilsTest {
+
+  @Test
+  public void testIt() {
+    for (int i = 1; i < 1000; i++) {
```

Comment:
Iterating 1000 times does not seem necessary since the current implementation is a naive one.